src/views/components/display-html/DisplayHTML.native.js

```diff
@@ -1,28 +1,21 @@
-import React, { Component } from 'react';
-import Html from 'react-native-render-html';
+import React from 'react';
+import HtmlPkg from 'react-native-render-html';
 import { string } from 'prop-types';
-import { Box } from '../../components';
 
-class DisplayHTML extends Component {
-  static defaultProps = {
-    content: '',
-  }
+const DisplayHTML = ({ Html }) =>  { 
```

Comment:
Oops - I meant make the component name `Html`, and the prop `html` (no capitalisation). Thanks!